source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -431,7 +431,8 @@ and `what's the point of periodic monitoring?`_)
 Client construction
 '''''''''''''''''''
 
-The client's constructor MUST NOT do any I/O.
+Except for the initial SRV lookup if given an SRV URI,
```

Comment:
Add TXT record lookup.

Reply:
I suggest we rephrase:

> Except for [initial DNS seedlist discovery](https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst) when given a connection string with a `mongodb+srv` scheme, the client's constructor MUST NOT perform any I/O."

The details about SRV/TXT records are not especially relevant to the SDAM spec. The important thing is that we're alluding to the DNS lookup behavior.

Reply:
I'll leave it to you to resolve this one @jmikola